### PR TITLE
DS-256 fix flex hero responsiveness buttton

### PR DESCRIPTION
**Pull request by bibifrost**

### CODE REVIEW

**Review by ThalieVideotron — COMMENTED** [review]
A1 pour moi

**Comment by ThalieVideotron** [comment]
je viens de voir qu'on a encore `Array<object>[]` ça nedevrait pas être `buttons[]` dans le fichier `hero.component.ts` ?

**Comment by JonathanLefebvreV** [comment]
Before:
image

After
image

---
Before:
image

After
image

---
Before
image

after
image

after with long text
image

**Review by JonathanLefebvreV — APPROVED** [review]
Other regression test have been one all-around story book

File: .storybook/preview.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ addParameters({
     controls: {
         expanded: true,
         hideNoControlsWarning: true,
+        sort: 'alpha',
```
Comment (JonathanLefebvreV) [inline .storybook/preview.ts:39]: By rebasing, this modif is not suppose to be there. 
Because it's only affect Storybook, It can be merge

**Pull request merged**